### DISCUSSION TITLE
[Support]: Frigate self reporting CPU and Memory levels not accurate?

### DISCUSSION THREAD

**Discussion by HomeAssistantNewbie** [blakeblackshear/frigate#13033] [General Support] [answered]
### Describe the problem you are having

Working on stabilizing and configuring frigate, but have noticed that the self reported RAM and CPU levels are quite different from those reported by HA and glances. Is it normal for it to be so deviated though? Frigate self reports around 15% memory usage, however glances attributes more like a 30% (1 to 1.2 GB of 3.8 GB). Same for CPU, where frigate self reports at maybe 30% and Glances attributes a 50% (although this one might be spikes), whereas RAM is pretty constant.

![image](https://github.com/user-attachments/assets/7b08a11c-a29c-46cb-9973-b43288399bda)
![image](https://github.com/user-attachments/assets/dffd6b3b-a490-41c4-a411-68eaa2a9a769)
![image](https://github.com/user-attachments/assets/52e8fa83-6cdb-4081-9314-11abecd912b7)

### Version

0.14.0

### Frigate config file

```yaml
Not applicable
```

### Relevant log output

```shell
![image](https://github.com/user-attachments/assets/7b08a11c-a29c-46cb-9973-b43288399bda)
```

### FFprobe output from your camera

```shell
Not applicable
```

### Operating system

HassOS

### Install method

HassOS Addon

### Object Detector

Coral

### Network connection

Wired

### Camera make and model

N/A

**Comment by NickM-27** [comment]
the first screenshot shows what specific pieces are using, so they can't be compared

**Reply by HomeAssistantNewbie** [reply]
Hi Nick, you mean the "general" tab is a "per camera"? Sorry, not understanding exactly

**Reply by NickM-27** [reply]
the first screenshot shows the CPU and memory usage of specific things in frigate, it seems you are comparing those to a general overview of glances but these are not comparable

**Reply by HomeAssistantNewbie** [reply]
Right, but I'm adding them up no? For memory: Detector Memory usage (not sure if this one is the coral's own memory or the one pulled from HAOS) 3.5%, go2rtc is 0.7%, logger 3.1%, recording 4% review 3%. So adding those up we get around 15%.

Unless you're saying there's other things not reported in metrics and these are just "key" values and the other 15% is just frigate doing other things?

**Reply by NickM-27** [reply, marked as answer]
you can't add them up, the % reported on the system metrics pages is % of a single CPU core meanwhile glances will show you the cumulative cpu usage. Also yes not every single thing is shown on the page

**Reply by HomeAssistantNewbie** [reply]
Right, from what I read frigate taking 1gb for 4 cameras total is normal so then closing off, thanks Nick!
